packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx

```diff
@@ -16,70 +16,15 @@ import {
   childrenWrapperStyle,
   childrenWrapperTransitionStyle,
   flagTextStyle,
+  iconStyle,
   iconThemeStyle,
   iconTransitionStyle,
   summaryHeader,
   summaryStyle,
   summaryTextThemeStyle,
   transitionDuration,
-} from './styles';
-
-/**
- * Types
- */
-interface ExpandableCardProps extends Omit<CardProps, 'contentStyle'> {
-  /**
-   * The title of the card
-   */
-  title: string;
-
-  /**
-   * Description text below the title
-   */
-  description?: ReactNode;
-
-  /**
-   * Text in parentheses immediately following the title. Typically 'optional' or 'required'
-   */
-  flagText?: 'optional' | 'required' | string;
-
-  /**
-   * Toggles dark mode
-   */
-  darkMode?: boolean;
-
-  /**
-   * Defines the default state of the card
-   */
-  defaultOpen?: boolean;
-
-  /**
-   * Forces the card state
-   */
-  isOpen?: boolean;
-
-  /**
-   * Callback fired when a user clicks the card header
-   */
-  onClick?: (
-    event: React.SyntheticEvent<HTMLDivElement, MouseEvent | KeyboardEvent>,
-  ) => void;
-
-  /**
-   * Unique id for the card
-   */
-  id?: string;
-
-  /**
-   * Styling prop for children
-   */
-  contentClassName?: string;
-
-  /**
-   * Component children
-   */
-  children?: React.ReactNode;
-}
+} from './ExpandableCard.styles';
+import { ExpandableCardProps } from './ExpandableCard.types';
```

Comment:
do we need to update the index file too?

Reply:
It was updated!